src/OpenTelemetry.Resources.OperatingSystem/OperatingSystemSemanticConventions.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+namespace OpenTelemetry.Resources.OperatingSystem;
+
+internal static class OperatingSystemSemanticConventions
+{
+    public const string AttributeOperatingSystemType = "os.type";
+
+    public static readonly string[] OperatingSystems =
```

Comment:
this does not look like is needed. If needed for tests, its best to declare it in tests project.

Reply:
I would say that you should define here separate values of the resource values.

You can see example here: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/49a01d3a8b72cbdba499d3bac302e055209eb89a/src/OpenTelemetry.SemanticConventions/Attributes/CloudAttributes.cs#L73-L81